src/commands/module/new.ts

```diff
@@ -62,14 +61,51 @@ export const generateNewModule = async (
   commandObj: CommandObj
 ): Promise<void> => {
   await fsPromises.mkdir(pathToModuleDir, { recursive: true });
-  const marineCli = await initMarineCli(commandObj);
   const name = path.basename(pathToModuleDir);
 
-  await marineCli({
-    command: "generate",
-    flags: { init: true, name },
-    workingDir: pathToModuleDir,
-  });
+  const newModuleSrcDirPath = path.join(pathToModuleDir, "src");
+  await fsPromises.mkdir(newModuleSrcDirPath, { recursive: true });
+
+  await fsPromises.writeFile(
+    path.join(newModuleSrcDirPath, "main.rs"),
+    MAIN_RS_CONTENT,
+    FS_OPTIONS
+  );
+
+  await fsPromises.writeFile(
+    path.join(pathToModuleDir, "Cargo.toml"),
+    getCargoTomlContent(name),
+    FS_OPTIONS
+  );
 
   await initNewReadonlyModuleConfig(pathToModuleDir, commandObj, name);
 };
+
+const MAIN_RS_CONTENT = `use marine_rs_sdk::marine;
+use marine_rs_sdk::module_manifest;
+
+module_manifest!();
+
+pub fn main() {}
+
+#[marine]
+pub fn greeting(name: String) -> String {
+    format!("Hi, {}", name)
+}
+`;
+
+const getCargoTomlContent = (name: string): string => `[package]
+name = "${name}"
+version = "0.1.0"
+edition = "2018"
+
+[[bin]]
+name = "${name}"
+path = "src/main.rs"
+
+[dependencies]
+marine-rs-sdk = "0.6.15"
```

Comment:
I think we should extract these versions to `const.ts` file

Reply:
Yeah, I should totally do it. Thanks